### PR TITLE
Add some samples

### PR DESCRIPTION
**Pull request by bricelam**
@natemcmaster @AlexanderTaeschner Is there anything I missed that you think would be good to cover?

### CODE REVIEW

File: samples/EncryptionSample/EncryptionSample.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.Data.Sqlite.Core\Microsoft.Data.Sqlite.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SQLitePCLRaw.bundle_sqlcipher" Version="1.1.8">
+      <!-- TODO: Add to Universe -->
```
Comment (bricelam) [inline samples/EncryptionSample/EncryptionSample.csproj]: @Eilon Can we get approval to use this package in a sample?

File: samples/AggregateFunctionSample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    {
+        static void Main()
+        {
+            var connection = new SqliteConnection("Data Source=:memory:");
+            connection.Open();
+
+            var createCommand = connection.CreateCommand();
+            createCommand.CommandText =
+            @"
+                CREATE TABLE student (
```
Comment (bricelam) [inline samples/AggregateFunctionSample/Program.cs]: I know it's unorthodox, but this seemed like the most readable format to me...

**Comment by natemcmaster** [comment]
LGTM. :shipit:

File: samples/EncryptionSample/EncryptionSample.csproj (hunk 1 above)
Comment (Eilon) [reply samples/EncryptionSample/EncryptionSample.csproj]: Approved!

File: samples/RegularExpressionSample/Program.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            var createCommand = connection.CreateCommand();
+            createCommand.CommandText =
+            @"
+                CREATE TABLE user (
+                    name TEXT,
+                    bio TEXT
+                );
+
+                INSERT INTO user
+                VALUES ('Arthur', 'I put two spaces after my full stops.  Always.'),
```
Comment (smitpatel) [inline samples/RegularExpressionSample/Program.cs]: @ajcvickers - :trollface:

Comment (bricelam) [reply samples/RegularExpressionSample/Program.cs]: This is a work of fiction. Names, characters, business, events and incidents are the products of the author’s imagination. Any resemblance to actual persons, living or dead, or actual events is purely coincidental.

File: samples/EncryptionSample/EncryptionSample.csproj (hunk 1 above)
Comment (bricelam) [reply samples/EncryptionSample/EncryptionSample.csproj]: Updated. YOLO pushed https://github.com/aspnet/Universe/commit/bf23f16c227ea9ba9b4eb0dfebe2d0e9e1140647

**Pull request merged**